gpcontrib/gpmapreduce/src/parse.c

```diff
@@ -15,9 +15,12 @@ int mapred_verify_object(mapred_parser_t *parser, mapred_object_t *obj);
 /* -------------------------------------------------------------------------- */
 int mapred_parse_error(mapred_parser_t *parser, char *fmt, ...)
 {
-	mapred_object_t *obj = parser->current_obj;
+	mapred_object_t *obj;
```

Comment:
```suggestion
	mapred_object_t *obj = NULL;
```
Since checks like:
https://github.com/arenadata/gpdb/blob/63fdc3071cabc5bec97805e929f4aaef782374bd/gpcontrib/gpmapreduce/src/parse.c#L34

Will be useless due to `obj` being initialized with garbage.